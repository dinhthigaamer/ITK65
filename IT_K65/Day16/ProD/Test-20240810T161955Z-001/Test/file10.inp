1000
89534479 495886262 310169570 995605514 478155606 790923835 -155784149 809537580 640950336 -229850993 516408631 -953573747 282412363 918138791 -16969938 -452181557 700205618 -997946533 967793995 884051683 -656298089 -821671496 812055459 861102134 58961278 711933494 -647137915 -873298079 881943456 -194293082 -872287309 -649079539 413346646 -332617126 986650770 143425343 -448711945 297472263 -100274836 721564494 379632885 -559818622 -423842271 -296083435 23897793 820268824 459322623 -354768928 353804154 -323585245 -829594027 432330421 -471561537 -460271886 345100402 231456783 445539900 529728842 -808170728 673157826 -70778491 642872105 299298867 -76674218 -902394063 377664752 723887448 -121522827 906464625 451793905 -646463035 -448092752 603587789 17153849 -453334835 -231418833 -749878480 -774813958 -834050815 -231570901 619441410 392226532 -215749231 -17192701 -688378779 -881309891 -877638425 -146878165 523758517 -158778747 -49535534 597311016 77325385 494128313 12111658 -4196724 295706874 477139483 375083042 726254399 -439255703 662913604 -481588088 673747948 -345999555 -723334478 -656721905 276267528 628773156 851420802 -999518145 488142826 931091653 804767279 -602535877 -298724038 836407150 -338005004 -256140383 -628450094 800577944 144987373 466219330 847405037 637321207 -447114130 181753508 -438748293 -235385103 -803907278 711868158 -362767003 749333691 -76248684 -185880588 -845591567 597144310 927726571 -137689278 -135021796 -338582594 827693356 558196441 -300735560 -682519635 645435942 -742977467 -506447125 -710531918 922105173 414824546 430974351 162495003 287520901 -348369119 -477335157 137176363 382926194 -554633865 872330669 -853878576 -677601317 -551034276 442712078 -102097764 144954079 -995229377 -859631407 -837281662 -398518468 -448902121 -402067283 -99027494 144069037 67022221 648353067 960373836 -478208687 275475644 817870738 -431655186 125292482 68149333 592748740 742519254 280244347 -874039352 -820647219 -928991342 -337110769 -743033709 -151795258 225430232 844291971 592728985 -159513162 -676118992 849731028 -277105922 882260594 -554079494 -594351928 -35130680 363384747 4084299 827562290 -735756735 210752217 -793109728 -468952006 177147525 396635340 -581182871 139977946 847846533 267844604 -748919536 983708594 -843737329 -17543219 337204546 -490630089 303973280 437483186 -236485208 -184889024 -735046675 492652841 -24670855 83024312 -737174430 -165439077 445360869 877812139 299521435 206398177 -509782444 894067066 -13694510 862964814 140834590 292014096 -23546991 874888561 -753301084 -439267342 -65167516 813358005 -581655705 -140318048 -944426104 37685100 963279968 -879096903 25655324 731698884 -886259080 -165799664 -669764515 -745782873 -493433335 -442904438 -832147253 990147559 -964285930 758568201 53948627 -575944040 -193228029 856210015 -800551604 950129602 -145754610 -622984052 -311905196 561408135 127064896 -906231444 -893350253 -674175873 -804573991 334443677 521002396 942028450 -71874753 217860245 -11028944 699481137 -60722358 396131830 219633742 -497199189 -611726003 429149320 -170859806 339882003 -761542422 607610642 249847939 587673237 -799980925 -79927239 -50597877 -933240486 707616382 -166732099 986951090 -762303091 844539884 -640982784 614977170 -724370701 -171122782 -319433218 -684306415 -182768362 16559683 362124143 269383176 -828108704 11291093 -539233794 -41548498 109564756 -83348343 77540126 -435840152 -679437516 -92655786 376020677 -800492489 934675575 104528445 865828634 210447924 991968920 640275663 37907647 337979076 -434716986 -622486375 47126470 -612683943 723436194 429728533 7428008 350783450 286870475 -847855660 -380908786 55194098 -899942993 -218085652 -836085487 -308322330 844874711 992821055 829220531 223403394 942256835 67467246 -390960188 202149758 429644393 -89802036 -40861317 -703093505 287363023 253990489 -706221891 288273657 64937262 -201056361 923942966 110308157 147510216 -718574933 -229792917 261907682 -93156378 -772651668 -138799817 362159212 -788337154 633192775 -332228420 316300467 762848198 -764109181 -356615357 -343656496 210316938 -180565604 865976778 536809823 -527853358 889888765 -372469067 381803021 -858444687 658151122 789919850 120432412 -928328565 451007112 358741890 325964751 -380541323 574142631 941967960 173653023 487592220 -832666939 -132511019 60933135 -465791058 -268024049 884622544 108800717 822316965 -485838427 -415365112 24881020 -16316930 456327300 -269933388 406874785 565714617 133279467 592351591 45988198 -386878428 -970085855 -112209973 476863429 -412333906 -256149175 999864530 -559427456 748357387 -851272606 405857869 -31946094 -374905782 259343293 -808880756 964466519 231522430 -145798030 -839850929 -681366641 284419075 -836350662 -662052153 -927012339 -385950619 -463446793 -14560551 752941303 832761265 359554640 -122606537 657316858 316232295 728414857 582530025 -625409269 -723898831 563691809 -663174302 974853189 -308783717 -678278109 203454367 -785682653 -292867446 422299316 -715128022 224831686 -678519116 -411663760 319895655 116626787 179630928 -749738235 -251685849 -558144715 395804911 -791837192 -78204749 -817240817 114609558 -855475745 720291962 393206663 -974952869 -107807552 353180249 994835640 733790219 -946432536 -585216394 363657077 672444973 549937988 340282332 -844602516 -633526758 139167798 739905158 999362555 -477835191 -918160467 80143122 -772703345 975938337 467415499 898149725 941240680 665439902 -977533928 -566443650 -445955570 569178659 -367190331 623389992 -827530553 -319746653 567998037 -183137877 582875726 -35046272 -988204811 40161456 -596008161 -723759531 297719353 -609353170 266487359 -582743757 -609099369 -872526432 387239440 138258771 -705217795 -480964015 68609157 340987484 858453240 -147256403 862912737 774948257 540272164 29726519 199238462 -629411090 303171801 146783313 87783684 364693380 11550471 859364548 70498910 -299375369 841400201 -389632931 836014665 -72779580 -211179922 583521776 319409714 700658981 -12320281 -258134206 656220649 -695775899 -418961295 -729695088 -269012014 832790587 537644456 878706025 -953810677 927159926 32109247 36646879 600884693 -552115125 973962320 -90989994 -766077535 14784359 -899879824 -991045768 104005729 232311600 -184522758 252638065 132799339 -635617899 -50067043 -613601351 670529791 -597990287 506108346 -895079704 -10282923 191606079 -506995919 31675932 887033015 767333935 884186390 200607771 437726771 978915213 69868769 442581436 -876448515 -275600508 467949355 407585032 346033897 -475711024 158296855 -267764302 616488013 553237468 -739932834 462700839 -518247050 -290031848 741311217 -655167159 -611784901 37487919 -436527780 -647523429 802683173 46398707 194110782 259368137 922907243 729262744 -185692948 -354057006 -141163075 -621549917 703563812 -943313929 334273341 -850257678 909483447 -497383248 371734083 -876514149 355977696 885777865 -427944665 -668814331 -768713863 540285115 -13849577 -980308615 228136832 625606346 -660105106 -664766499 411641852 -633892324 -218043256 717257938 -822210037 546809128 -100218668 335885830 978518755 847262096 -420127728 -68711387 242483584 768462461 747519045 478922882 546657842 -258479124 -519379363 -95802718 652485714 308643420 -831265695 -267763216 887712417 -493699143 207142894 -620910290 -769502137 631184106 45536802 414961547 -569964928 747803905 75860325 -615146042 -605337972 845998328 865241377 -636266206 158460252 -111163320 369465090 303477850 548751972 -588088593 450241186 -606501753 382274050 -876991361 -452793704 -992101037 -358094766 474868740 -509868515 -134801079 -229504648 849769478 -594021727 -98901782 -707296153 922048770 432457131 -611300123 -668011410 595219477 -908700439 -319829353 -926449152 64923909 673696568 668044575 -135425348 -286950111 813080868 540834665 781200167 -976715323 594113481 601602437 299060023 894935137 390477022 22604426 -222428105 37368274 -892793379 707397183 960141568 -684693007 471686725 553375569 -819116853 320589140 731033971 331191712 893022997 27458494 -49157329 -228105874 -276064836 -506816404 323725710 217061374 627494476 -53221097 -266812723 214144348 698566252 820184816 505057792 356114234 -306111769 -90075833 308854765 -444898587 400529828 532443602 -884387333 -436300311 982289958 -95663502 391795384 257928993 -791215825 643924724 -968351470 -679958452 -397292130 -579437531 -159078885 833259102 44571727 616552632 -993020955 -769780309 506560200 860741270 263015726 -931898784 -387420087 -603751997 -707718351 -252399063 -76100110 718335508 63620573 577660901 550703717 691789512 -201012452 -775794642 -458612729 973987972 217295701 66987558 199017741 -441952297 632700116 122342007 -159725936 -593569335 -271855077 -244896582 -472927242 529559380 -89279792 381122961 896727814 -441121 -254877402 869837829 -338180884 -850858408 931866522 207890547 -906896979 601129990 373191332 150799733 -772745689 -97788823 886849384 959624942 -930022719 -930366951 -949568905 -727731795 -268957983 -224157081 532624770 646587203 -431284723 -118332810 166193760 497464647 -530013817 9592446 -313565510 -978515222 571891924 -98732979 18897680 482526647 -771560187 -448487226 -574946103 -477356142 -590662689 907628638 924335345 595226713 -529670538 44495846 -245756740 191087126 -860080121 -93015364 -761841540 519330480 -90509483 -510027038 739077943 -719247287 -543179241 208905101 205046623 728222550 -380345048 952010872 716015790 -816748779 -787027905 445456255 229966045 890525904 -646930512 -560500315 -882863965 -830346889 -194401370 -345772683 547093462 -710982669 -639669960 -275364133 -190669788 -974721255 -808602220 336179149 -884856060 82116103 -592100425 -701534557 -675051029 -422342292 -899857507 -147130044 993615422 -186744766 -342568340 -349940824 -894613227 517216817 -795106355 530545009 -943822064 -31627069 -145132514 -487825766 -173600869 250193667 139669305 -217514108 235635706 542890477 618587636 -263487333 -755863733 -292900156 72278930 630034897 19057336 -909300258 -659057029 -798406072 405294592 502335110 629055847 698504112 -221865960 -314873789 291289311 243022746 -793179191 -690257506 -757509766 -984891447 -625576658 -455237656 287728170 95041316 21977202 23641188 920899729 321838708 516096766 311627962 -370834933 103609566 767284151 873455 446217167 -848535049 -285787711 -239392230 -721183505 863118529 -64332016 666642285 -827031817 -812194500 -122424534 619493674 -245772587 -798049191 -361499126 -756795541 -229001171 -875427384 126731891 346605577 
89534479 495886262 310169570 995605514 478155606 790923835 -155784149 809537580 640950336 -229850993 516408631 -953573747 282412363 918138791 -16969938 -452181557 700205618 -997946533 967793995 884051683 -656298089 -821671496 812055459 861102134 58961278 711933494 -647137915 -873298079 881943456 -194293082 -872287309 -649079539 413346646 -332617126 986650770 143425343 -448711945 297472263 -100274836 721564494 379632885 -559818622 -423842271 -296083435 23897793 820268824 459322623 -354768928 353804154 -323585245 -829594027 432330421 -471561537 -460271886 345100402 231456783 445539900 529728842 -808170728 673157826 -70778491 642872105 299298867 -76674218 -902394063 377664752 723887448 -121522827 906464625 451793905 -646463035 -448092752 603587789 17153849 -453334835 -231418833 -749878480 -774813958 -834050815 -231570901 619441410 392226532 -215749231 -17192701 -688378779 -881309891 -877638425 -146878165 523758517 -158778747 -49535534 597311016 77325385 494128313 12111658 -4196724 295706874 477139483 375083042 726254399 -439255703 662913604 -481588088 673747948 -345999555 -723334478 -656721905 276267528 628773156 851420802 -999518145 488142826 931091653 804767279 -602535877 -298724038 836407150 -338005004 -256140383 -628450094 800577944 144987373 466219330 847405037 637321207 -447114130 181753508 -438748293 -235385103 -803907278 711868158 -362767003 749333691 -76248684 -185880588 -845591567 597144310 927726571 -137689278 -135021796 -338582594 827693356 558196441 -300735560 -682519635 645435942 -742977467 -506447125 -710531918 922105173 414824546 430974351 162495003 287520901 -348369119 -477335157 137176363 382926194 -554633865 872330669 -853878576 -677601317 -551034276 442712078 -102097764 144954079 -995229377 -859631407 -837281662 -398518468 -448902121 -402067283 -99027494 144069037 67022221 648353067 960373836 -478208687 275475644 817870738 -431655186 125292482 68149333 592748740 742519254 280244347 -874039352 -820647219 -928991342 -337110769 -743033709 -151795258 225430232 844291971 592728985 -159513162 -676118992 849731028 -277105922 882260594 -554079494 -594351928 -35130680 363384747 4084299 827562290 -735756735 210752217 -793109728 -468952006 177147525 396635340 -581182871 139977946 847846533 267844604 -748919536 983708594 -843737329 -17543219 337204546 -490630089 303973280 437483186 -236485208 -184889024 -735046675 492652841 -24670855 83024312 -737174430 -165439077 445360869 877812139 299521435 206398177 -509782444 894067066 -13694510 862964814 140834590 292014096 -23546991 874888561 -753301084 -439267342 -65167516 813358005 -581655705 -140318048 -944426104 37685100 963279968 -879096903 25655324 731698884 -886259080 -165799664 -669764515 -745782873 -493433335 -442904438 -832147253 990147559 -964285930 758568201 53948627 -575944040 -193228029 856210015 -800551604 950129602 -145754610 -622984052 -311905196 561408135 127064896 -906231444 -893350253 -674175873 -804573991 334443677 521002396 942028450 -71874753 217860245 -11028944 699481137 -60722358 396131830 219633742 -497199189 -611726003 429149320 -170859806 339882003 -761542422 607610642 249847939 587673237 -799980925 -79927239 -50597877 -933240486 707616382 -166732099 986951090 -762303091 844539884 -640982784 614977170 -724370701 -171122782 -319433218 -684306415 -182768362 16559683 362124143 269383176 -828108704 11291093 -539233794 -41548498 109564756 -83348343 77540126 -435840152 -679437516 -92655786 376020677 -800492489 934675575 104528445 865828634 210447924 991968920 640275663 37907647 337979076 -434716986 -622486375 47126470 -612683943 723436194 429728533 7428008 350783450 286870475 -847855660 -380908786 55194098 -899942993 -218085652 -836085487 -308322330 844874711 992821055 829220531 223403394 942256835 67467246 -390960188 202149758 429644393 -89802036 -40861317 -703093505 287363023 253990489 -706221891 288273657 64937262 -201056361 923942966 110308157 147510216 -718574933 -229792917 261907682 -93156378 -772651668 -138799817 362159212 -788337154 633192775 -332228420 316300467 762848198 -764109181 -356615357 -343656496 210316938 -180565604 865976778 536809823 -527853358 889888765 -372469067 381803021 -858444687 658151122 789919850 120432412 -928328565 451007112 358741890 325964751 -380541323 574142631 941967960 173653023 487592220 -832666939 -132511019 60933135 -465791058 -268024049 884622544 108800717 822316965 -485838427 -415365112 24881020 -16316930 456327300 -269933388 406874785 565714617 133279467 592351591 45988198 -386878428 -970085855 -112209973 476863429 -412333906 -256149175 999864530 -559427456 748357387 -851272606 405857869 -31946094 -374905782 259343293 -808880756 964466519 231522430 -145798030 -839850929 -681366641 284419075 -836350662 -662052153 -927012339 -385950619 -463446793 -14560551 752941303 832761265 359554640 -122606537 657316858 316232295 728414857 582530025 -625409269 -723898831 563691809 -663174302 974853189 -308783717 -678278109 203454367 -785682653 -292867446 422299316 -715128022 224831686 -678519116 -411663760 319895655 116626787 179630928 -749738235 -251685849 -558144715 395804911 -791837192 -78204749 -817240817 114609558 -855475745 720291962 393206663 -974952869 -107807552 353180249 994835640 733790219 -946432536 -585216394 363657077 672444973 549937988 340282332 -844602516 -633526758 139167798 739905158 999362555 -477835191 -918160467 80143122 -772703345 975938337 467415499 898149725 941240680 665439902 -977533928 -566443650 -445955570 569178659 -367190331 623389992 -827530553 -319746653 567998037 -183137877 582875726 -35046272 -988204811 40161456 -596008161 -723759531 297719353 -609353170 266487359 -582743757 -609099369 -872526432 387239440 138258771 -705217795 -480964015 68609157 340987484 858453240 -147256403 862912737 774948257 540272164 29726519 199238462 -629411090 303171801 146783313 87783684 364693380 11550471 859364548 70498910 -299375369 615093326 -389632931 836014665 -72779580 -211179922 583521776 319409714 700658981 -12320281 -258134206 656220649 -695775899 -418961295 -729695088 -269012014 832790587 537644456 878706025 -953810677 927159926 32109247 36646879 600884693 -552115125 973962320 -90989994 -766077535 14784359 -899879824 -991045768 104005729 232311600 -184522758 252638065 132799339 -635617899 -50067043 -613601351 670529791 -597990287 506108346 -895079704 -10282923 191606079 -506995919 31675932 887033015 767333935 884186390 200607771 437726771 978915213 69868769 442581436 -876448515 -275600508 467949355 407585032 346033897 -475711024 158296855 -267764302 616488013 553237468 -739932834 462700839 -518247050 -290031848 741311217 -655167159 -611784901 37487919 -436527780 -647523429 802683173 46398707 194110782 259368137 922907243 729262744 -185692948 -354057006 -141163075 -621549917 703563812 -943313929 334273341 -850257678 909483447 -497383248 371734083 -876514149 355977696 885777865 -427944665 -668814331 -768713863 540285115 -13849577 -980308615 228136832 625606346 -660105106 -664766499 411641852 -633892324 -218043256 717257938 -822210037 546809128 -100218668 335885830 978518755 847262096 -420127728 -68711387 242483584 768462461 747519045 478922882 546657842 -258479124 -519379363 -95802718 652485714 308643420 -831265695 -267763216 887712417 -493699143 207142894 -620910290 -769502137 631184106 45536802 414961547 -569964928 747803905 75860325 -615146042 -605337972 845998328 865241377 -636266206 158460252 -111163320 369465090 303477850 548751972 -588088593 450241186 -606501753 382274050 -876991361 -452793704 -992101037 -358094766 474868740 -509868515 -134801079 -229504648 849769478 -594021727 -98901782 -707296153 922048770 432457131 -611300123 -668011410 595219477 -908700439 -319829353 -926449152 64923909 673696568 668044575 -135425348 -286950111 813080868 540834665 781200167 -976715323 594113481 601602437 299060023 894935137 390477022 22604426 -222428105 37368274 -892793379 707397183 960141568 -684693007 471686725 553375569 -819116853 320589140 731033971 331191712 893022997 27458494 -49157329 -228105874 -276064836 -506816404 323725710 217061374 627494476 -53221097 -266812723 214144348 698566252 820184816 505057792 356114234 -306111769 -90075833 308854765 -444898587 400529828 532443602 -884387333 -436300311 982289958 -95663502 391795384 257928993 -791215825 643924724 -968351470 -679958452 -397292130 -579437531 -159078885 833259102 44571727 616552632 -993020955 -769780309 506560200 860741270 263015726 -931898784 -387420087 -603751997 -707718351 -252399063 -76100110 718335508 63620573 577660901 550703717 691789512 -201012452 -775794642 -458612729 973987972 217295701 66987558 199017741 -441952297 632700116 122342007 -159725936 -593569335 -271855077 -244896582 -472927242 529559380 -89279792 381122961 896727814 -441121 -254877402 869837829 -338180884 -850858408 931866522 207890547 -906896979 601129990 373191332 150799733 -772745689 -97788823 886849384 959624942 -930022719 -930366951 -949568905 -727731795 -268957983 -224157081 532624770 646587203 -431284723 -118332810 166193760 497464647 -530013817 9592446 -313565510 -978515222 571891924 -98732979 18897680 482526647 -771560187 -448487226 -574946103 -477356142 -590662689 907628638 924335345 595226713 -529670538 44495846 -245756740 191087126 -860080121 -93015364 -761841540 519330480 -90509483 -510027038 739077943 -719247287 -543179241 208905101 205046623 728222550 -380345048 952010872 716015790 -816748779 -787027905 445456255 229966045 890525904 -646930512 -560500315 -882863965 -830346889 -194401370 -345772683 547093462 -710982669 -639669960 -275364133 -190669788 -974721255 -808602220 336179149 -884856060 82116103 -592100425 -701534557 -675051029 -422342292 -899857507 -147130044 993615422 -186744766 -342568340 -349940824 -894613227 517216817 -795106355 530545009 -943822064 -31627069 -145132514 -487825766 -173600869 250193667 139669305 -217514108 235635706 542890477 618587636 -263487333 -755863733 -292900156 72278930 630034897 19057336 -909300258 -659057029 -798406072 405294592 502335110 629055847 698504112 -221865960 -314873789 291289311 243022746 -793179191 -690257506 -757509766 -984891447 -625576658 -455237656 287728170 95041316 21977202 23641188 920899729 321838708 516096766 311627962 -370834933 103609566 767284151 873455 446217167 -848535049 -285787711 -239392230 -721183505 863118529 -64332016 666642285 -827031817 -812194500 -122424534 619493674 -245772587 -798049191 -361499126 -756795541 -229001171 -875427384 126731891 346605577 