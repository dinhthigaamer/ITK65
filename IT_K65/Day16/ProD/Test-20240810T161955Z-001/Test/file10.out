-6154372690672
